composer.json

```diff
@@ -1,33 +1,35 @@
 {
-    "name": "php-pm/php-pm",
```

Comment:
Because this change has nothing to do with the actual PR and will varnish the blaming history.